assets/data/datasets.csv

```diff
@@ -0,0 +1,48 @@
+Name;Network Focus;Host Focus;Start Year;End Year;Setting;OS Type;Network Log Source;Network Logs Labeled;Host Log Source;Host Logs Labeled;Attack Categories;User Emulation;Packed Size in MB;Unpacked Size in MB
```

Comment:
- It is unclear to me what "focus" means. Does it make sense to use "Network Data" and "Host Data" instead?
- Network Log Source -> Network Data Sources
- Network Logs Labeled -> Network Data Labeled
- Host Log Source -> Host Data Sources
- Host Logs Labeled -> Host Data Labeled
- User Emulation -> Benign Activity